resources/istio/values.yaml

```diff
@@ -1,6 +1,7 @@
 ---
 helmValues:
   global:
+    autoscalingv2API: true
```

Comment:
We shouldn't add this as this is marked for internal usage only ([ref](https://github.com/istio/istio/pull/37872#discussion_r847543725)). This field is set depending on the configuration in the IOP.

Reply:
Good point, confirmed it is working